graphql_client/src/lib.rs

```diff
@@ -288,6 +290,8 @@ pub struct Response<Data> {
     pub data: Option<Data>,
     /// The top-level errors returned by the server.
     pub errors: Option<Vec<Error>>,
+    /// The top-level errors returned by the server.
```

Comment:
The doc string should probably say something else. Ideally a short explanation with a link to the spec.